core/GenerisRdf.php

```diff
@@ -31,6 +31,7 @@ interface GenerisRdf
     public const PROPERTY_ALIAS = 'http://www.tao.lu/Ontologies/generis.rdf#alias';
     public const PROPERTY_IS_LG_DEPENDENT = 'http://www.tao.lu/Ontologies/generis.rdf#is_language_dependent';
     public const PROPERTY_IS_STATISTICAL = 'http://www.tao.lu/Ontologies/generis.rdf#is_statistical';
+    public const PROPERTY_IS_EDITABLE = 'http://www.tao.lu/Ontologies/generis.rdf#is_editable';
```

Comment:
Why we need this to correct that "some" properties are not synched to datastore?

Reply:
We need this one to mark lists as non-editable for local lists in UI.